lncfg/gossip.go

```diff
@@ -9,7 +9,7 @@ import (
 
 //nolint:lll
 type Gossip struct {
-	PinnedSyncersRaw []string `long:"pinned-syncers" description:"A set of peers that should always remain in an active sync state, which can be used to closely synchronize the routing tables of two nodes. The value should be comma separated list of hex-encoded pubkeys. Connected peers matching this pubkey will remain active for the duration of the connection and not count towards the NumActiveSyncer count."`
+	PinnedSyncersRaw []string `long:"pinned-syncers" description:"A set of peers that should always remain in an active sync state, which can be used to closely synchronize the routing tables of two nodes. The value should be a hex-encoded pubkey, the flag can be specified multiple times to add multiple peers. Connected peers matching this pubkey will remain active for the duration of the connection and not count towards the NumActiveSyncer count."`
```

Comment:
Given that this is parsed as follows
```golang
for _, pubkeyStr := range g.PinnedSyncersRaw
```
did it even work with comma separated values?

Reply:
No, it never worked with comma separated values. Since the code was all added in the same PR I assume it was only ever tested with a single value.